pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -34,10 +34,25 @@ type Code int
 // NodeScoreList declares a list of nodes and their scores.
 type NodeScoreList []NodeScore
 
+func (n NodeScoreList) Len() int {
+	return len(n)
+}
+
+func (n NodeScoreList) Less(i, j int) bool {
+	if n[i].Score == n[j].Score {
+		return n[i].Name < n[j].Name
+	}
+	return n[i].Score < n[j].Score
+}
+
+func (n NodeScoreList) Swap(i, j int) {
+	n[i], n[j] = n[j], n[i]
+}
```

Comment:
why do we need those methods?

Reply:
I recalled some tests need to "sort" NodeScoreList - before `HostPriorityList` supports that.

Reply:
We can do something like this for tests: ```sort.Slice(out, func(i, j int) bool { return out[i].Score < out[j].Score })```

Reply:
Done.